src/lib.rs

```diff
@@ -533,6 +533,7 @@ mod tests {
     use crate::{
         body::{Edge, State},
         header::Header,
+        output::to_hoa,
         AcceptanceAtom, AcceptanceCondition, AcceptanceName, AcceptanceSignature, Body, HeaderItem,
         HoaAutomaton, Label, StateConjunction, ALPHABET, VARS,
```

Comment:
I'm not sure how to run the tests acturally, I can't find ALPHABET and VARS defined anywhere in this reop.